### DISCUSSION TITLE
Portable version Windows?

### DISCUSSION THREAD

**Discussion by carloselectro** [jeffvli/sonixd#379] [Q&A]
Hi, Is there any way to save the app settings the app directory, so it can be portable accross Windows?

**Comment by jeffvli** [comment]
The user settings lives inside the `C:\Users\$your_user\Appdata\Roaming\Sonixd\` folder (or `%appdata%\Sonixd`.

**Comment by carloselectro** [comment]
Thanks, I know about %appdata%\Sonixd, I was hoping for a way to run the app so it saves the settings in the app directory instead of the roaming profile.
